File: Latihan_A/Disk.cpp
class Disk
{
    private:
        // atribut
        string type;
        string capacity;
        int price;

    public:
        // constructor
        Disk(){}
        Disk(string type, string capacity, int price) {
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        // method set untuk type, capacity, dan price
        void setType(string type) {
            this->type = type;
        }
        void setCapacity(string capacity) {
            this->capacity = capacity;
        }
        void setPrice(int price) {
            this->price = price;
        }
        
        // method get untuk type, capacity, dan price
        string getType() {
            return this->type;
        }
        string getCapacity() {
            return this->capacity;
        }
        int getPrice() {
            return this->price;
        }

        // method output untuk Disk
        void printDisk() {
            cout << "Disk" << endl;
            cout << "-------------" << endl;
            cout << "Type        : " << this->type << endl;
            cout << "Capacity    : " << this->capacity << endl;
            cout << "Price       : " << "Rp. " << this->price << endl;
        }

        // destructor
        ~Disk(){}
};

File: Latihan_B/python/Disk.py
class Disk:
    # atribut private
    __type = ""
    __capacity = ""
    __price = 0

    # constructor
    def __init__(self, type="", capacity="", price=0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # method set untuk type, capacity, dan price
    def setType(self, type):
        self.__type = type
    def setCapacity(self, capacity):
        self.__capacity = capacity
    def setPrice(self, price):
        self.__price = price

    # method get untuk type, capacity, dan price
    def getType(self):
        return self.__type
    def getCapacity(self):
        return self.__capacity
    def getPrice(self):
        return self.__price
    
    # method untuk output Disk
    def printDisk(self):
        print("Disk")
        print("-------------")
        print("Type        : " + self.getType())
        print("Capacity    : " + self.getCapacity())
        print("Price       : " + "Rp. " + str(self.getPrice()))